.github/workflows/compiler-warnings.yml

```diff
@@ -57,7 +59,11 @@ jobs:
 
     - name: Install dependencies
       run: |
-        .github/workflows/scripts/install-build-deps.sh
```

Comment:
Looks like with `gcc` package and `ubuntu-latest`, it's not gcc-11, but gcc-13. Not a big deal, but the job name might be missleading.